### DISCUSSION TITLE
Update release of SG-NEx dataset

### DISCUSSION THREAD

**Discussion by cying111** [GoekeLab/sg-nex-data#34] [Announcements]
We are happy to announce the update release (v0.4.0) of the SG-NEx data! 

We have added 
- raw signal data in BLOW5 format, 
- with a tutorial on usage [here](https://github.com/GoekeLab/sg-nex-data/blob/master/docs/SG-NEx_blow5_tutorial.md). 

For how to access the data on AWS, you can refer to the data access tutorial [here](https://github.com/GoekeLab/sg-nex-data/blob/master/docs/SG-NEx_blow5_tutorial.md) or you can directly browse the S3 bucket with the BLOW5 files [here](http://sg-nex-data-blow5.s3-website-ap-southeast-1.amazonaws.com/)

<hr /><em>This discussion was created from the release <a href='https://github.com/GoekeLab/sg-nex-data/releases/tag/v0.4.0'>Update release of SG-NEx dataset</a>.</em>

**Comment by lpryszcz** [comment]
do you plan to add `pod5` as well? it seems it's becoming official format for ONT data...

**Reply by jonathangoeke** [reply]
not immediately, but at some point the fast5 could be replaced with pod5 files. BLOW5 to pod5 conversion might be an alternative as well
